Sources/PackageModel/Target.swift

```diff
@@ -329,6 +329,20 @@ public final class SwiftTarget: Target {
         )
     }
 
+    public init(name: String, type: Target.Kind? = nil, dependencies: [Target.Dependency], testEntryPointSources sources: Sources) {
+        self.swiftVersion = .v5
+
+        super.init(
+            name: name,
+            type: type ?? .executable,
+            path: .root,
+            sources: sources,
+            dependencies: dependencies,
+            buildSettings: .init(),
+            pluginUsages: []
+        )
+    }
+
```

Comment:
could this helper initializer move somewhere more local to the call site, so it is not part of the public API?

Reply:
Done